controllers/util/provider.go

```diff
@@ -47,6 +48,10 @@ const (
 	EnvARMSubscriptionID = "ARM_SUBSCRIPTION_ID"
 	EnvARMTenantID       = "ARM_TENANT_ID"
 
+	EnvVSphereUser               = "VSPHERE_USER"
+	EnvVSpherePassword           = "VSPHERE_PASSWORD"
+	EnvVSphereServer             = "VSPHERE_SERVER"
+	EnvVSphereAllowUnverifiedSSL = "VSPHERE_ALLOW_UNVERIFIED_SSL"
```

Comment:
As you are trying to support the optional argument `VSPHERE_ALLOW_UNVERIFIED_SSL`, will you consider supporting `VSPHERE_VIM_KEEP_ALIVE`?

Reply:
I’m not sure about this .I'm just getting started with vsphere.  This parameter is supported only because there is it in the example.

Reply:
You can reference to https://registry.terraform.io/providers/hashicorp/vsphere/latest/docs#argument-reference.

> allow_unverified_ssl - (Optional) Boolean that can be set to true to disable SSL certificate verification. This should be used with care as it could allow an attacker to intercept your auth token. If omitted, default value is false. Can also be specified with the VSPHERE_ALLOW_UNVERIFIED_SSL environment variable.

> vim_keep_alive - (Optional) Keep alive interval in minutes for the VIM session. Standard session timeout in vSphere is 30 minutes. This defaults to 10 minutes to ensure that operations that take a longer than 30 minutes without API interaction do not result in a session timeout. Can also be specified with the VSPHERE_VIM_KEEP_ALIVE environment variable.